airbyte-integrations/connectors/source-ashby/source_ashby/manifest.yaml

```diff
@@ -18,7 +18,7 @@ definitions:
         type: SimpleRetriever
         requester:
           $ref: "#/definitions/base_requester"
-          path: /applications.list
+          path: /application.list
```

Comment:
You need to bump the metadata, pyproject and doc changelog entry with a new version.

Reply:
Done!